src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -114,7 +115,10 @@ public static CodeWriter.CodeWriterScope WriteMethodDeclaration(this CodeWriter
                 writer.Append($": base(");
                 foreach (var parameter in method.BaseMethod.Parameters)
                 {
-                    writer.Append($"{parameter.Name:I}, ");
+                    if (parameter is ParameterInvocation invocation && invocation.Invocation != null)
```

Comment:
nit use braces when if statement has an else or if statement is more than 1 line